app/controllers/test/saml_test_controller.rb

```diff
@@ -111,5 +111,13 @@ def test_saml_settings
       settings.idp_slo_target_url = "http://#{Figaro.env.domain_name}/api/saml/logout"
       settings
     end
+
+    def saml_client_private_key
+      @saml_client_private_key ||= File.read("#{Rails.root}/config/saml_client_private_key.enc")
```

Comment:
That's a good point. I hadn't realized that this is only used by `SamlTestController`. However, this isn't just for specs. `SamlTestController` allows us to test the SAML flow in development by visiting the `/test/saml` endpoint in the browser.

What I think I will do is create a new `keys` folder, and move `saml.key.enc` there, and rename `saml_client_private_key` to `saml_test.key.enc` or something like that.